React/Views/RCTTabBar.h

```diff
@@ -9,6 +9,9 @@
 
 #import <UIKit/UIKit.h>
 
+#import "RCTComponent.h"
```

Comment:
I don't recall making this change... might have been there when I started the Apple TV work and just was carried along with my commits.  I'll remove this.